### PR TITLE
Added an integration test for ckeditor/ckeditor5#1265

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Tests: Added an integration test for ckeditor/ckeditor5#1265.

Requires: https://github.com/ckeditor/ckeditor5-engine/pull/1559.

### CODE REVIEW

File: tests/table-integration.js [hunk 1]
```diff
@@ -78,6 +79,18 @@ describe( 'Table feature – integration', () => {
 				[ '<blockQuote><paragraph>bar[]</paragraph></blockQuote>' ]
 			] ) );
 		} );
+
+		it.only( 'merges elements without throwing errors', () => {
```
Comment (Reinmar) [inline tests/table-integration.js]: `.only()`

Comment (ma2ciek) [reply tests/table-integration.js]: We should have a test whether some test conains `/(describe|it)\.only\(/` on the CI 😄

Comment (pomek) [reply tests/table-integration.js]: Coverage drops below the "critical" level. It should be a warning for us. Unfortunately, @Reinmar was faster then I read the email :D

File: tests/manual/table.js [hunk 2]
```diff
@@ -9,10 +9,11 @@ import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor'
 import ArticlePluginSet from '@ckeditor/ckeditor5-core/tests/_utils/articlepluginset';
 import Table from '../../src/table';
 import TableToolbar from '../../src/tabletoolbar';
+import BlockQuote from '@ckeditor/ckeditor5-block-quote/src/blockquote';
 
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
-		plugins: [ ArticlePluginSet, Table, TableToolbar ],
+		plugins: [ ArticlePluginSet, Table, TableToolbar, BlockQuote ],
```
Comment (Reinmar) [inline tests/manual/table.js]: Why was this change made? If you want to have a playground to test table with block quotes we'd also need this button in the toolbar.

Comment (pomek) [reply tests/manual/table.js]: image

The button is already added. https://github.com/ckeditor/ckeditor5-table/pull/129/files#diff-4097acfafae638fcb639ff3a9d047f3cR18

Comment (pomek) [reply tests/manual/table.js]: Yea, I was also a little surprised. I wanted to revert my invalid change than I saw the button :D

**Pull request merged**